src/main/resources/bin/windup

```diff
@@ -57,7 +57,7 @@ while [ "$1" != "" ] ; do
     WINDUP_DEBUG_ARGS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8000"
   fi
 
-  QUOTED_ARGS="$QUOTED_ARGS \"$1\""
+  QUOTED_ARGS="$QUOTED_ARGS $1"
```

Comment:
Can you remind me why all the quotes need to be taken out of these lines? I can't really follow the tangled web of quoting in here to figure out if this will break anything.